src/kernel/boot.c

```diff
@@ -553,6 +553,24 @@ BOOT_CODE tcb_t *create_initial_thread(cap_t root_cnode_cap, cap_t it_pd_cap, vp
     return tcb;
 }
 
+#ifndef clock_sync_test
+BOOT_CODE void clock_sync_test(void)
+{
+    ticks_t t, t0;
+
+    t = NODE_STATE_ON_CORE(ksCurTime, 0);
```

Comment:
Maybe add an `assert(0 != getCurrentCPUIndex())` here, as this must be called on secondary cores only, otherwise it deadlocks.